ny-portal/src/app/(frontend)/komponenter/page.tsx

```diff
@@ -8,12 +9,22 @@ export default async function Components() {
 
     const data = await payload.find({
         collection: "component-page",
+        select: {
+            title: true,
+            slug: true,
+        },
     });
 
     return (
-        <ul>
+        <ul style={{ display: "flex", flexDirection: "column", gap: "1rem" }}>
             {data.docs.map((page) => (
-                <li key={page.title}>{page.title}</li>
+                <Link
+                    style={{ color: "var(--jkl-color-text-default)" }}
+                    key={page.slug}
+                    href={`/komponenter/${page.slug}`}
+                >
+                    {page.title}
+                </Link>
```

Comment:
Innser nå at her bør vi jo bruke Jøkul sin `Link`-komponent, nå som den er polymorf. Det gjorde vi jo nettopp for å kunne bruke lenkekomponenter fra rammeverk som Next enkelt. 

Hvis du f.eks. endrer til `import NextLink from "next/link"`, og legger til `import { Link } from "@fremtind/jokul/components/link"`, kan du får riktig styling fra Jøkul slik:

```suggestion
                <Link
                    as={NextLink}
                    key={page.slug}
                    href={`/komponenter/${page.slug}`}
                >
                    {page.title}
                </Link>
```